querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -184,3 +187,52 @@ export const QueryEngineSelector: React.FC<IQueryEngineSelectorProps> = ({
         <div className="QueryEngineSelector">{engineButtonDOM}</div>
     );
 };
+
+const QueryLimitSelector: React.FC<{
+    rowLimit: number;
+    setRowLimit: (rowLimit: number) => void;
+    tooltipPos: TooltipDirection;
+}> = ({ rowLimit, setRowLimit, tooltipPos }) => {
+    const rowLimitOptions = React.useMemo(
+        () =>
+            ROW_LIMIT_SCALE.map((value) => ({
+                label: formatNumber(value),
+                value,
+            })),
+        []
```

Comment:
does it need to put `ROW_LIMIT_SCALE` into the dependency array? it can be changed from the console, right?

otherwise, this `rowLimitOptions` wont change and the below `useEffect` doesn't need it as dependency either.

Reply:
nah, its intended as an constant. it also can't be changed from console, thats the plugin value you are thinking about.

I took out rowLimitOptions to be a module constant. thanks!